eng/Packages.Data.props

```diff
@@ -97,6 +97,7 @@
     <PackageReference Update="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
     <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <PackageReference Update="System.Threading.Channels" Version="4.7.1" />
+    <PackageReference Update="System.Security.Cryptography.Cose" Version="7.0.0" />
     <PackageReference Update="System.Security.Cryptography.ProtectedData" Version="4.7.0" />
```

Comment:
```suggestion
    <PackageReference Update="System.Security.Cryptography.Cose" Version="7.0.0" />
    <PackageReference Update="System.Security.Cryptography.ProtectedData" Version="4.7.0" />
    <PackageReference Update="System.Threading.Channels" Version="4.7.1" />
    <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.4" />    
```

Reply:
Not introduced by this PR, but the new package is in just the right place to see that someone messed up the alphabetical ordering.